filters.json.example

```diff
@@ -288,5 +288,270 @@
         "Lugia":"False",
         "Ho-Oh":"False",
         "Celebi":"False"
+    },
+    "raids":{
+        "enabled":"True",
+        "filters": {
+            "min_level": "1",
+            "max_level": "5",
+            "ignore_eggs": "False"
+        },
+        "default": {
+            "min_dist":"0", "max_dist":"inf", "min_cp": "0", "max_cp": "999999", "min_iv":"0", "max_iv":"100",
+            "min_atk": "0", "max_atk":"15", "min_def": "0", "max_def":"15", "min_sta": "0", "max_sta":"15",
+            "quick_move": null, "charge_move": null, "moveset": null,
+            "size": null, "gender": null, "form": null, "ignore_missing": "False"
+        },
+        "Bulbasaur": "True",
```

Comment:
We should get rid of the Pokemon that aren't currently used in raids. When/if Pokemon get rotated we can update the example or people can change their configs accordingly.